view/theme/frio/templates/nav.tpl

```diff
@@ -1,311 +1,421 @@
 {{* we have modified the navmenu (look at function frio_remote_nav() ) to have remote links. $nav.userinfo is a new variable and replaces the original $userinfo variable *}}
 {{if $nav.userinfo}}
-<header>
-	{{* {{$langselector}} *}}
+    <header>
+        {{* {{$langselector}} *}}
```

Comment:
Templates are indented using tabs, please tweak your formatter to not convert to spaces instead.

Reply:
Damn nice catch!

Reply:
I think VS Code doesn't understand Smarty files and doesn't take the ruleset.xml rules into consideration. I also don't know why it would ignore the `.editorconfig` file.